src/configLoader.js

```diff
@@ -13,12 +14,18 @@ export function configLoader(configPath) {
         return process.exit(1);
     }
 
+    const rawConfig = fs.readFileSync(configPath, 'utf8');
+    const compiledConfig = rawConfig.replace(/\$\$\$_([A-Za-z]+)_\$\$\$/g, (match, variableName) => {
```

Comment:
Can we start using a more natural environment variable syntax than `$$$`? Maybe just `${var}`?

Reply:
Also I think we should support a slightly larger character set for the actual variable names. `[A-Za-z0-9_]+` perhaps?